testData/dummy_repo/pants

```diff
@@ -3,100 +3,191 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 # =============================== NOTE ===============================
-# This pants bootstrap script comes from the pantsbuild/setup
+# This ./pants bootstrap script comes from the pantsbuild/setup
 # project and is intended to be checked into your code repository so
 # that any developer can check out your code and be building it with
-# pants with no prior setup needed.
+# Pants with no prior setup needed.
 #
-# You can learn more here: https://pantsbuild.github.io/setup
+# You can learn more here: https://www.pantsbuild.org/install.html
 # ====================================================================
 
-set -e
+set -eou pipefail
 
-PYTHON=${PYTHON:-$(which python2.7)}
+PYTHON_BIN_NAME="${PYTHON:-unspecified}"
 
-PANTS_HOME="${PANTS_HOME:-${HOME}/.cache/pants/setup}"
+PANTS_HOME="${PANTS_HOME:-${XDG_CACHE_HOME:-$HOME/.cache}/pants/setup}"
```

Comment:
I'm very curious about how we ran into the need to specify `XDG_CACHE_HOME` here :) (not saying it's bad, it's a good change, I'm just very curious

Reply:
@blorente Thanks for the review! Sorry I didn't provide enough context. The script was from the installation command from https://www.pantsbuild.org/install.html. Do you think we can land as is?

Reply:
The related code is under https://github.com/pantsbuild/setup/blob/gh-pages/pants

Reply:
Oh cool, yeah :) that makes sense.